test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -395,6 +397,7 @@ public void getStudyParticipant() {
         List<ConsentSignature> sigs1 = Lists.newArrayList(new ConsentSignature.Builder()
                 .withName("Name 1").withBirthdate("1980-01-01").build());
         when(account.getConsentSignatureHistory(SubpopulationGuid.create("guid1"))).thenReturn(sigs1);
+        when(account.getClientData()).thenReturn(TestUtils.getClientData());
```

Comment:
For organizational purposes, this should probably be moved up to the rest of the mock statements.

Reply:
Yes... moved alongside the other account mock statements.